src/utilities/getConfig.ts

```diff
@@ -0,0 +1,22 @@
+import { readFile } from 'node:fs/promises';
+import { findUp } from 'find-up';
+
+export async function getConfig(): Promise<sernConfig> {
+	const sernLocation = await findUp('sern.config.json');
+
+	if (!sernLocation) throw new Error("Can't find sern.config.json");
+
+	const output = JSON.parse(await readFile(sernLocation, 'utf8')) as sernConfig;
+
+	if (!output) throw new Error("Can't read your sern.config.json.");
```

Comment:
ive had the habit of using assert more for checking undefined.

its declarative and reads better

```ts
import assert from 'node:assert'

assert(output, "Can't read your sern.config.json.")
```